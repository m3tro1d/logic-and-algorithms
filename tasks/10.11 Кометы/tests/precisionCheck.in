2 100
-5 -5
-1 100
