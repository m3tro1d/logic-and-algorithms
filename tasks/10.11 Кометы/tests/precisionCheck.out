0.0000099795
